examples/notebook/src/commands.ts

```diff
@@ -396,17 +396,17 @@ export const setupCommands = (
       command: COMMAND_IDS.findPrevious
     },
     {
-      selector: '.jp-Notebook.jp-mod-commandMode :focus:not(:read-write)',
+      selector: '.jp-Notebook.jp-mod-commandMode:not(.jp-mod-readWrite) :focus',
```

Comment:
Question: should we keep the extra `not(:read-write)`? No functional difference, but if user downgrades to 4.1.0 for some reason this would be compatible.

Reply:
Not sure, since both versions are documented in the extension migration guide.